src/apis/types.ts

```diff
@@ -265,6 +271,104 @@ export interface ChampionSummaryDto {
 
 export type LaneSummaryDto = Record<Position, number>;
 
+export interface MatchBriefRes {
+  matchInfo: MatchDto;
+  participant: ParticipantDto;
+  allParticipants: ParticipantBriefDto[];
+}
+
+interface MatchDto {
+  matchId: string;
+  version: string;
+  gameStartAt: Date;
+  gameEndAt: Date;
```

Comment:
```suggestion
  gameStartAt: string;
  gameEndAt: string;
```
시간 관련 정보는 문자열로 오는 걸로 알고 있습니다.